src/Npgsql/NpgsqlReadBuffer.cs

```diff
@@ -148,6 +149,12 @@ async Task EnsureLong()
                 catch (Exception e)
                 {
                     Connector.Break();
+
+                    if (e is OperationCanceledException)
+                    {
+                        e = new TimeoutException("Timeout during reading attempt");
+                    }
```

Comment:
nit: There is no need in braces for oneliners.

Reply:
Done